devops/helm/clamAV/_clamav/values.yaml

```diff
@@ -72,9 +72,8 @@ hpa:
   # requests: 1k
 
 podDisruptionBudget:
-  enabled: false
-  # minAvailable: 1
-  # maxUnavailable: 1
+  enabled: true
+  minAvailable: 2
```

Comment:
Since the replica count is also set to 2, should it not be 1 to allow one POD to be moved?